libkbfs/block_retrieval_worker_test.go

```diff
@@ -61,20 +62,15 @@ func (bg *fakeBlockGetter) getBlock(ctx context.Context, kmd KeyMetadata, blockP
 		return errors.New("Block doesn't exist in fake block map")
 	}
 	// Wait until the caller tells us to continue
-	select {
-	case <-source.ch:
-		bytes, err := bg.codec.Encode(source.block)
-		if err != nil {
-			return err
-		}
-		err = bg.codec.Decode(bytes, block)
-		if err != nil {
-			return err
+	for {
+		select {
+		case source.startCh <- struct{}{}:
```

Comment:
I'm confused as to why this works.  Don't we need to do the `startCh` (if possible), and then wait for the `continueCh`?  With this construction, it seems like we'd do the start without the continue, or vice versa.